src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
                     "term": "EU DCC",
                     "anchor": "eu_dcc",
                     "description": "Siehe <a href='#glossary_dcc'>DCC</a>"
+                },
+                {
+                    "term": "European Federation Gateway Service (EFGS)",
+                    "anchor": "efgs",
+                    "description": "Eine zentrale Infrastruktur, welche dazu dient <a href='#glossary_diagnosis_key'>Diagnoseschlüssel</a> über Ländergrenzen hinweg auszutauschen und zu prüfen."
+                },
+                {
+                    "term": "Event Registrierung",
+                    "anchor": "event_registration",
+                    "description": "Registrieren für eine Veranstaltung oder einen Besuch eines Ortes."
+                },
+                {
+                    "term": "Exposure Notification Express (ENE)",

```

Comment:
Also for Android. See comment in English FAQ.

Reply:
Links to FAQ article now, thanks!